insecure/corruptible/helper.go

```diff
@@ -13,6 +14,18 @@ type mockAttackerObserveClient struct {
 	closed         chan interface{}
 }
 
+func (m *mockAttackerObserveClient) SetHeader(_ metadata.MD) error {
+	panic("not implemented")
```

Comment:
what's the reason for these methods not being implemented?

Reply:
this is a mock implementation used solely for testing, the methods are there to satisfy a gRPC interface. Though most of those methods are not used as part of the test. Added more clarification in https://github.com/onflow/flow-go/pull/2599/commits/87298533b8aa8c61a544556d87d1451ce501e1f9.